update_registry_final.py

```
@@ -0,0 +1,34 @@
+    if '[0024]' in line:
+        count_0024 += 1
+        if count_0024 == 2:
+            line = line.replace('0024-concept-expiration-ttl.md', '0024-performance-optimizations-phase2.md')
+    if '[0030]' in line:
+        line = line.replace('0030-cli-crate-architecture.md', '0030-test-and-benchmark-gap-remediation.md')
+    new_lines.append(line)
+
+# Add ADR-0049 if missing from table
+found_49 = False
+for line in new_lines:
+    if '[0049]' in line: found_49 = True
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)